app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/integration/VpnNetworkStackVariantStore.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.mobile.android.vpn.integration
+
+import androidx.core.content.edit
+import com.duckduckgo.mobile.android.vpn.prefs.VpnSharedPreferencesProvider
+
+interface VpnNetworkStackVariantStore {
+    var variant: String?
+}
+
+internal class VpnNetworkStackVariantStoreImpl constructor(
+    sharedPreferencesProvider: VpnSharedPreferencesProvider
+) : VpnNetworkStackVariantStore {
+
+    private val preferences = sharedPreferencesProvider.getSharedPreferences(FILE_PREFERENCES, multiprocess = true, migrate = false)
```

Comment:
Making it multi process because we'll probably need to check this in the browser process as well (eg pixels)